core/src/main/scala/com/salesforce/op/stages/impl/regression/RegressionModelSelector.scala

```diff
@@ -199,6 +199,7 @@ object RegressionModelsToTry extends Enum[RegressionModelsToTry] {
   case object OpRandomForestRegressor extends RegressionModelsToTry
   case object OpGBTRegressor extends RegressionModelsToTry
   case object OpGeneralizedLinearRegression extends RegressionModelsToTry
+  case object OpXGBoostRegressor extends RegressionModelsToTry
```

Comment:
please also define some default grid settings for this to run in regression

Reply:
also for the other model selectors

Reply:
ok if that is the plan then lots hold off on adding it to the enum as well - particularly since you only added it to regression and not multiclass and binary

Reply:
sounds good. removed.